db_test.go

```diff
@@ -1494,6 +1494,40 @@ func TestIndexQueryWithSort(t *testing.T) {
 	})
 }
 
+func checkUniqueTimestamps(db *c.DB) {
+	// Retrieve all documents from the collection
```

Comment:
Let's remove `checkUniqueTimestamps`, it sounds something useful only for debugging during development.